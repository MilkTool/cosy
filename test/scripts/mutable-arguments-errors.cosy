
{
    mut sss = 12
    fn mutable_param2(mut s2 Num) {
        s2 = 666
        print s2
    }
    mutable_param2(mut sss)
    print sss
}
