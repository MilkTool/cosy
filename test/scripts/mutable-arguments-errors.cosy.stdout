[line 4] Error at "s2": Only struct parameters can be marked as `mut`.
